ocsp_status.go

```diff
@@ -45,17 +42,14 @@ func checkFromCert(certFile string, isPost bool, ocspURL string, dir string, has
 		return fmt.Errorf("Error getting issuer certificate from certificate: %w", err)
 	}
 
-	ocspReq, err := ocsptools.CreateOCSPReq(ocspURL, leafCert, issuerCert, reqMethod, hash)
+	ocspResp, err := ocsptools.FetchOCSPResp(ocspURL, dir, leafCert, issuerCert, reqMethod, hash)
 	if err != nil {
-		return fmt.Errorf("Error creating OCSP Request: %w", err)
+		return fmt.Errorf("Error fetching OCSP response: %w", err)
 	}
 
-	ocspResp, err := ocsptools.GetOCSPResponse(ocspReq)
-	if err != nil {
-		return fmt.Errorf("Error getting OCSP Response: %w", err)
-	}
+	linter.LintOCSPResp(ocspResp)
 
-	return ocsptools.ParseAndLint(ocspResp, issuerCert)
+	return nil
```

Comment:
Is return nil required at the end of a function? Will go return nil by default like how Python returns None by default?

Reply:
I think return nil is required at the end